package.json

```diff
@@ -51,16 +53,12 @@
     "cross-env": "^4.0.0",
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.15.1",
-    "eslint": "^3.6.0",
-    "eslint-config-airbnb": "^13.0.0",
```

Comment:
airbnb version 18 has some good rules, and I want them, but it would be a major version bump to support them. Maybe later